modules/36_buildings/services_with_capital/datainput.gms

```diff
@@ -34,13 +34,16 @@ pm_delta_kap(regi,in)$nests_putty_dyn36(out,in) = pm_delta_kap(regi,out);
       
 parameter
 
-p36_floorspace_scen                    "floorspace"
+p36_floorspace_scen(tall,all_regi,all_demScen)                  "floorspace"
```

Comment:
out of curiosity: the previous notion just loaded the data without any check on the dimensions?

Reply:
in case of p36_floorspace_scen: it was also declared in declarations.gms with set-dimensions. But for some other parameters I now added the first time the set-dimension.